src/backend/gl/src/command.rs

```diff
@@ -1088,7 +1087,9 @@ impl command::RawCommandBuffer<Backend> for RawCommandBuffer {
     }
 
     unsafe fn dispatch_indirect(&mut self, buffer: &n::Buffer, offset: buffer::Offset) {
-        self.push_cmd(Command::DispatchIndirect(buffer.raw, offset));
+        let (raw_buffer, range) = buffer.borrow().as_bound();
+        assert_eq!(range.start, 0, "buffer offset unsupported in indirect draw");
```

Comment:
we actually support indirect buffer offset, it's a part of `DispatchIndirect` variant.
What we do *not* support is `draw_indexed_indirect` when index buffer offset is non-zero

Reply:
Right, okay that makes sense, I was confused between `DispatchIndirect` and `draw_indirect` and `draw_indexed_indirect`.  OpenGL doesn't support `draw_indexed_indirect` at all right now so that's not an issue.